echopype/calibrate/api.py

```diff
@@ -53,7 +53,21 @@ def _compute_cal(
     )
     # Perform calibration
     if cal_type == "Sv":
-        return cal_obj.compute_Sv(waveform_mode=waveform_mode, encode_mode=encode_mode)
+
+        if "range_offset" in echodata.platform.data_vars.keys():
+
+            # add range_offset to the created xr.Dataset
+            sv_dataset = cal_obj.compute_Sv(
+                waveform_mode=waveform_mode, encode_mode=encode_mode
+            )
+            sv_dataset["range_offset"] = echodata.platform.range_offset
+            return sv_dataset
+
+        else:
+            return cal_obj.compute_Sv(
+                waveform_mode=waveform_mode, encode_mode=encode_mode
+            )
+
```

Comment:
This block looks correct. But I'd like to suggest the more compact alternative below. It's your call, though.

```python
if cal_type == "Sv":
    sv_dataset = cal_obj.compute_Sv(
        waveform_mode=waveform_mode, encode_mode=encode_mode
    )

    if "range_offset" in echodata.platform.data_vars.keys():
        # add range_offset to the created xr.Dataset
        sv_dataset["range_offset"] = echodata.platform.range_offset

    return sv_dataset
```

Reply:
I second this :)